src/main/resources/configCivcraft.yml

```diff
@@ -7216,7 +7216,7 @@ recipes:
         durability: 3
       eggs:
         material: EGG
-        amount: 8
+        amount: 4
```

Comment:
With eggs being nothing but an afk resource from a chicken grinder, I would suggest just completly removing them from the recipe for now and possibly adding a replacement.